<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gestionmateriel</class>
 <widget class="QDialog" name="Gestionmateriel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>981</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane { /* The tab widget frame */
border-top: 2px solid #2a2a2a; 
border-left: 2px solid #2a2a2a;
border-right: 2px solid #2a2a2a ; 
border-bottom: 2px solid #2a2a2a; 
background: #dcdcdc;
	
	
	/*border-top-color: rgb(0, 0, 0);*/

}
QTabWidget::tab-bar {
left: 0px; /* move to the right by 5px */
}
QTabBar::tab {
background: #dcdcdc;
border: 2px solid #535353;
border-bottom-color: #535353; 
border-top-left-radius: 7px;
border-top-right-radius: 7px;
min-width: 80px;
padding: 8px;
color: rgb(50, 50, 162);
	color: rgb(0, 0, 0);
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:selected,QTabBar::tab:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
}
QTabBar::tab:selected {
border-color: #9B9B9B;
border-bottom-color: #535353; /* same as pane color */
color: #dcdcdc;
font-size:13px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>CRUD</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>811</width>
       <height>421</height>
      </rect>
     </property>
     <property name="title">
      <string>Gestion Materiels</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFFF00;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFFF00;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>Qualite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFFF00;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>Prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="qualite">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prix">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFFF00;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>Date d'acaht</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="date_achat">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>56</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFFF00;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
      </property>
      <property name="text">
       <string>Chercher l'ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/projet c++/o.g/wp4390809.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_8</zorder>
    <zorder>groupBox</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Métiers</string>
    </attribute>
    <widget class="QTableView" name="table_materiel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>731</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
     </property>
     <property name="text">
      <string>Trie par ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
     </property>
     <property name="text">
      <string>Trie par Nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
     </property>
     <property name="text">
      <string>Trie par Prix</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
     border: 1px solid #bac4e0;
     border-radius: 3px;
     padding: 1px 18px 1px 3px;
     min-width: 6em;
 }
 
 QComboBox:editable {
     background: #bac4e0;
 }
 
 QComboBox:!editable, QComboBox::drop-down:editable {
      background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                   stop: 0 #d5ddf7, stop: 1 #b0c2f7);
 }
 
 QComboBox::drop-down {
     subcontrol-origin: padding;
     subcontrol-position: top right;
     width: 23px;
 
     border-left-width: 1px;
     border-left-color: #bac4e0;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
 
 QComboBox::down-arrow {
     image: url(C:/Users/Administrateur/Downloads/Atelier_Connexion/icon.png);
 }</string>
     </property>
     <item>
      <property name="text">
       <string>Rechercher par ID</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rechercher par nom</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rechercher par prix</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
     border: 2px solid #6374a6;
     border-radius: 10px;
     padding: 0 8px;
     background: #dce3f7;
 }</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
     </property>
     <property name="text">
      <string>rechercher</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>260</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>410</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius: 8px;
color: rgb(255,255,0);
background-color: rgba(239, 0, 0);
border: 2px solid ;
}
QPushButton:hover
{
color: rgb(239, 0, 0);
background-color: rgb(255,255,0);
}</string>
     </property>
     <property name="text">
      <string>imprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>210</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: #000000;
background:#FFFF00;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}</string>
     </property>
     <property name="text">
      <string>Générer la liste des materiels en PDF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>360</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: #000000;
background:#FFFF00;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}</string>
     </property>
     <property name="text">
      <string>Imprimer la liste des materiels</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>D:/projet c++/o.g/Diving-Rio_2016_Olympic_Games_HD_Vector_Wallpaper_1366x768.jpg</pixmap>
     </property>
    </widget>
    <zorder>label_9</zorder>
    <zorder>table_materiel</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>pushButton_10</zorder>
    <zorder>pushButton_11</zorder>
    <zorder>comboBox</zorder>
    <zorder>lineEdit</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>pushButton_7</zorder>
    <zorder>pushButton_8</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
